src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -6981,6 +7013,34 @@ FlowAnalysisAnnotations getRValueAnnotations(BoundExpression expr)
             }
         }
 
+        private static FlowAnalysisAnnotations GetPropertySetterAnnotations(PropertySymbol property)
```

Comment:
>GetPropertySetterAnnotations [](start = 47, length = 28)

Consider moving below `GetLValueAnnotations()` to reduce the differences in the PR.